Sources/Logging.php

```diff
@@ -260,6 +260,7 @@ function displayDebug()
 
 	if (!empty($modSettings['cache_enable']) && !empty($cache_hits))
 	{
+		$missed_entires = array();
```

Comment:
it's actually broken in the original code too. might be best to just fix it to avoid confusion.